src/ua/hillel/Dog.java

```diff
@@ -0,0 +1,29 @@
+public class Dog extends Animals{
+     int max_run =500;
```

Comment:
The same here, please rename all variables in this class according to Camel case conventions.
Make them all private.